docs/api/apiv3/paths/storage_files.yml

```diff
@@ -0,0 +1,51 @@
+# /api/v3/storages/{id}/files
+---
+get:
+  summary: Gets files of a storage.
+  operationId: get_storage_files
+  tags:
+    - File links
+  description: |-
+    Gets a collection of file data from a storage. 
+    
+    If no `root` context was given, the result is the content of the document root. With `root` context given, the
+    result contains the collections of file data from without the given root file id.
+    
+    If given `root` context is no directory, but a file, the collection contains this file data as a single element. 
+  parameters:
+    - name: id
+      description: Storage id
+      in: path
+      required: true
+      schema:
+        type: integer
+      example: 1337
+    - name: root
+      description: Root file id
+      in: query
+      required: false
+      schema:
+        type: integer
```

Comment:
As discussed. this should be a string so storage providers other than Nextcloud which may not use numerical IDs work too.